crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -646,6 +646,8 @@ pub enum RethRpcModule {
     Web3,
     /// `rpc_` module
     Rpc,
+    /// `ots_` module
+    Ots,
```

Comment:
minor: probably worth adding to `test_rpc_module_str`: https://github.com/paradigmxyz/reth/blob/62153b07896117e8651f6bc38e4138f6386fe40a/crates/rpc/rpc-builder/src/lib.rs#L1748C8-L1748C27